firefox-ios/firefox-ios-tests/Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -369,7 +369,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
                 // There is no Cancel option in iPad.
                 app.otherElements["PopoverDismissRegion"].tap()
             } else {
-                app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].tapOnApp()
+                app.buttons[AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.closeButton].tap()
```

Comment:
iOS 15's Tracking Protection menu occupies the whole screen. `tapOnApp` does not close the menu.